data-loader/data/v2/schema.graphql

```diff
@@ -22,3 +22,19 @@ type Property {
   type: String @dgraph(pred: "type") @search(by: [exact])
   value: String @dgraph(pred: "value") @search(by: [exact])
 }
+
+type ConfigurationItem {
+  id: ID!
+  code: String! @id @dgraph(pred: "code") @search(by: [exact, trigram])
+  name: String @dgraph(pred: "name") @search(by: [exact, fulltext, trigram])
+  type: String @dgraph(pred: "type") @search(by: [exact])
+  children: [ConfigurationItem] @dgraph(pred: "subItems")
+  parents: [ConfigurationItem] @dgraph(pred: "~subItems")
+}
+
+type PropertyConfigurationItem {
```

Comment:
This isn't implemented in this PR, if we want to make it configurable, we'll need to create another PR...